update infos about static horizontal parameters (#321)
MeteoSwiss/opendata: docs/e-forecast-data/e2-e3-numerical-weather-forecasting-model.mdx

Reviewer: https://github.com/MeteoSwiss/opendata/pull/321/files#diff-14ec5a0416b37b4b6692b2bc45b593d093e55f3b9443cba6e1eb336d8a8f9bafR151 
The title should be changed as the files also contain surface properties now, e.g. "Accessing constants: Height, latitude, longitude, and surface properties". If doing so, also check the references to that section e.g. https://github.com/MeteoSwiss/opendata/blob/0b4765e78dc765365865dfc21d048396e37b84ed/docs/e-forecast-data/e2-e3-numerical-weather-forecasting-model.mdx?plain=1#L397 which need to be updated

    > 💡 **Tip for new users**: We recommend inexperienced GRIB file users to take a look at the provided [Jupyter Notebooks](https://github.com/MeteoSwiss/opendata-nwp-demos). The data retrieval with the Python API includes fetching longitude and latitude.

    Besides the current forecast files, each collection contains two static files: the vertical constants file, which stores information about the heights of the half-levels (HHL) in the vertical grid, and the horizontal constants file, which contains the following parameters.
    - CLON: The longitude of the center point coordinate of each triangle on the horizontal grid
    - CLAT: The latitude of the center point coordinate of each triangle on the horizontal grid
    - FR_LAND: The land cover (1 if there is land, 0 if there is the sea)
    - HSURF: Geometric height of the earths surface above sea level
    - SOILTYPE: Type of soil according to the [soilType.table](https://github.com/COSMO-ORG/eccodes-cosmo-resources/blob/master/definitions/soilType.table)

>     #### Accessing horizontal grid parameters

Reviewer: ```suggestion
    #### Accessing horizontal constants

```

    ```
    wget -O <desired_filename> “<pre-signed URL>”
    ```

> 
>     Once the static GRIB file is downloaded, you may want to associate a value from a data file (for a given parameter) to its latitude and longitutde coordinate.
>     To achieve this, ensure that the `uuidOfHGrid` (Universally Unique Identifier for the horizontal grid) key in the data file matches the one in the parameters `CLON` and `CLAT` of the the static horizontal file.
>     Then, add the associated coordinates to the values from the data file on a point-by-point basis.

Reviewer: I would avoid adding the part on how to assign the coordinates to the grid points. And the warning can be generalized to all parameters in the file not only lat/lon
```suggestion

Before using any constants from the static file, verify that its `uuidOfHGrid` (Universally Unique Identifier for the horizontal grid) GRIB key matches the one in your forecast data. This ensures the horizontal grid definitions are consistent across files.
```

    ```
    wget -O <desired_filename> “<pre-signed URL>”
    ```

> Before using any constants from the static file, verify that its `uuidOfHGrid` (Universally Unique Identifier for the horizontal grid) GRIB key matches the one in your forecast data. This ensures the horizontal grid definitions are consistent across files.

Reviewer: Indentation 
```suggestion
    Before using any constants from the static file, verify that its `uuidOfHGrid` (Universally Unique Identifier for the horizontal grid) GRIB key matches the one in your forecast data. This ensures the horizontal grid definitions are consistent across files.

```
